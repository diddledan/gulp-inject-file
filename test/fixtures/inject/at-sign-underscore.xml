<?xml version="1.0" encoding="UTF-8"?>
<data>
	<existing />
	<!-- inject: ./dirwith@sign/sub_item.xml -->
</data>
