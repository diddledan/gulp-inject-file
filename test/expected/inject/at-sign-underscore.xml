<?xml version="1.0" encoding="UTF-8"?>
<data>
	<existing />
	<sub-item1 />
	<sub-item2 />
</data>
